telemetry/definitions/commonDefinitions.json

```diff
@@ -3581,32 +3551,6 @@
                 }
             ]
         },
-        {
-            "name": "codeTransform_vcsViewerCanceled",
-            "description": "An interactivity measured when users cancel a patch view.",
-            "metadata": [
-                {
-                    "type": "codeTransformJobId",
-                    "required": false
-                },
-                {
-                    "type": "codeTransformPatchViewerCancelSrcComponents",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformRuntimeError",
-                    "required": false
-                },
-                {
-                    "type": "codeTransformSessionId",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformStatus",
-                    "required": true
-                }
-            ]
-        },
         {
             "name": "codeTransform_vcsViewerClicked",
```

Comment:
Get rid of `vcsViewerCanceled` and `vcsViewerSubmitted`; repurpose `vcsViewerClicked` to fill in for these 2 removals.